electron/renderer/src/components/App.jsx

```diff
@@ -37,6 +38,11 @@ class App extends React.Component {
   componentDidMount() {
     window.addEventListener(EVENT_TYPE.ACTION.SWITCH_ACCOUNT, this.switchAccount, false);
     window.addEventListener(EVENT_TYPE.ACTION.CREATE_SSO_ACCOUNT, this.initiateSSO, false);
+
+    // Workaround: Switch to first webview after startup
+    setTimeout(() => {
+      this.switchAccount({detail: {accountIndex: 0}});
+    }, 1000);
```

Comment:
Do we have other chances than setting a timeout?

Reply:
I haven't found another solution yet